packages/runtime/runtime-utils/src/summaryUtils.ts

```diff
@@ -93,7 +97,7 @@ function calculateStatsCore(summaryObject: SummaryObject, stats: ISummaryStats):
     }
 }
 
-export function calculateStats(summary: ISummaryTree): ISummaryStats {
+export function calculateStats(summary: SummaryObject): ISummaryStats {
```

Comment:
Is this change still needed with all the recent changes?

Reply:
Let me check

Reply:
Yes, it's used here
```
            const gcSummaryTreeStats = summaryTree.tree[gcTreeKey]
                ? calculateStats(summaryTree.tree[gcTreeKey])
                : undefined;
```